dask/dataframe/io/tests/test_parquet.py

```diff
@@ -2731,6 +2731,20 @@ def test_optimize_and_not(tmpdir):
         assert_eq(a, b)
 
 
+@pytest.mark.parametrize("dtype", ["str", "bool", "float"])
```

Comment:
Just wondering what prompted the addition of testing against multiple `dtype`s. Is there some relevant logic for empty DataFrames with different dtypes?

Reply:
Good question! I think I was conflating this issue with #8373, I can take this part out